manager/PROJECT

```diff
@@ -13,6 +13,9 @@ resources:
 - group: app
   kind: Blueprint
   version: v1alpha1
+- group: app
```

Comment:
Did you really use kubebuilder? I thought it would error. Anyway, if you did then please also fix the rest of this file because it says admin instead of app. Otherwise let's just remove this file.

Reply:
I did not use kubebuilder, just thought this file is needed for 'make generate'. I will delete it.